crypto/fipsmodule/aes/asm/aesni-xts-avx512.pl

```diff
@@ -1460,7 +1460,10 @@
 
   my $rndsuffix = &random_string();
 
-  $code .= ".text\n";
+  $code .= <<___;
+#ifndef MY_ASSEMBLER_IS_TOO_OLD_FOR_AVX
```

Comment:
This is the start of code generation for the `if` block that started way up at [line 49](https://github.com/aws/aws-lc/blob/main/crypto/fipsmodule/aes/asm/aesni-xts-avx512.pl#L49)